pkg/schedule/config/config.go

```diff
@@ -57,6 +57,7 @@ const (
 	defaultEnablePlacementRules  = true
 	defaultEnableWitness         = false
 	defaultHaltScheduling        = false
+	defaultEnableSchedulePlugin  = false
```

Comment:
Maybe we need to add a release note about the default value changed

Reply:
added.